translations/es/LC_MESSAGES/messages.po

```diff
@@ -103,15 +103,15 @@ msgstr "Página de Aterrizage AMP"
 #: /content/support/faqs/supported-platforms@es.md
 #: /content/support/faqs/supported-platforms@ko.md
 msgid "AMP Overview"
-msgstr ""
+msgstr "Descripción general de AMP"
```

Comment:
The word "overview" does not translate to "descripción general".
In spanish, Overview is something else, but if you want to said that., oK.